plenum/server/consensus/ordering_service.py

```diff
@@ -1665,7 +1674,8 @@ def l_tryOrder(self, commit: Commit):
     def l_doOrder(self, commit: Commit):
         key = (commit.viewNo, commit.ppSeqNo)
         self._logger.debug("{} ordering COMMIT {}".format(self, key))
-        return self.l_order_3pc_key(key)
+        self._bus.send(TryOrderMsg(self._data.inst_id, key, self.l_getPrePrepare(*key)))
+        # return self.l_order_3pc_key(key)
```

Comment:
I think we need to call `l_order_3pc_key` here instead of sending TryOrderMsg to the bus